packaging/setup/plugins/ovirt-engine-setup/ovirt-engine-keycloak/provisioning/postgres.py

```diff
@@ -127,6 +128,8 @@ def _customization(self):
         self.environment[
             okkcons.ProvisioningEnv.POSTGRES_PROVISIONING_ENABLED
         ] = self._enabled = enabled
+        if self._enabled:
```

Comment:
We need to move this here, before CUSTOMIZATION, because the way this works is:
- getCredentials loops over all the things it needs to get.  If any are missing, it asks for them.
- applyEnvironment sets all needed env keys to the values it's going to eventually use (supplied user/db, random password).
- So getCredentials misses nothing and asks for nothing.

We have this in a few other places as well.